gcloud/bigtable/row.py

```diff
@@ -227,6 +227,80 @@ def to_pb(self):
         return data_pb2.RowFilter(column_qualifier_regex_filter=self.regex)
 
 
+class ColumnRangeFilter(RowFilter):
+    """A row filter to restrict to a range of columns.
+
+    Both the start and end column can be included or excluded in the range.
+    By default, we include them both, but this can be changed with optional
+    flags.
+
+    :type column_family_id: str
+    :param column_family_id: The column family that contains the columns. Must
+                             be of the form ``[_a-zA-Z0-9][-_.a-zA-Z0-9]*``.
+
+    :type start_column: bytes
+    :param start_column: The start of the range of columns. If no value is
+                         used, it is interpreted as the empty string
+                         (inclusive) by the backend.
```

Comment:
Apply the [same change suggested for `start_value`](https://github.com/GoogleCloudPlatform/gcloud-python/pull/1309#discussion_r48190753) in #1309?